cdap-docs/admin-manual/source/installation/security.rst

```diff
@@ -248,12 +260,16 @@ Basic Authentication
 The simplest way to identity a client is to authenticate against a realm file.
 To configure basic authentication add the following properties to ``cdap-site.xml``:
 
-========================================================== ==================================================================
-Property                                                   Value
-========================================================== ==================================================================
-``security.authentication.handlerClassName``               ``co.cask.cdap.security.server.BasicAuthenticationHandler``
-``security.authentication.basic.realmfile``                ``<path>`` *(either absolute or relative)*
-========================================================== ==================================================================
+========================================================== =========================== ======================================
+Property                                                   Default Value               Description
```

Comment:
Should we use actual configuration file snippets here instead of the table format?  Maybe it would be clearer if we said something like:

"To configure basic authentication using a realm file, add the following properties to cdap-site.xml:
```
<property>
  <name>security.authentication.handlerClassName</name>
  <value>co.cask.cdap.security.server.BasicAuthenticationHandler</value>
</property>
<property>
  <name>security.authentication.basic.realmfile</name>
  <value>/path/to/realm/file</value>
</property>
```

Isn't that closer to the format that we would use elsewhere?